newdle/client/src/components/answer/Calendar.js

```diff
@@ -182,7 +183,9 @@ export default function Calendar() {
   const busyTimes = useSelector(getBusyTimes);
   const activeDate = toMoment(useSelector(getActiveDate), HTML5_FMT.DATE);
   const dispatch = useDispatch();
+
   const defaultHourSpan = MAX_HOUR - MIN_HOUR;
+  const isTabletOrMobile = useMediaQuery({query: '(max-width: 1224px)'});
```

Comment:
You use this in multiple places, so maybe wrap it in a custom `useIsSmallScreen` hook so we don't have the media query string in multiple places.

Reply:
and for the failing lint check you could just call the hook again there instead of passing it as a prop